js/setup.js

```diff
@@ -0,0 +1,99 @@
+'use strict';
+
+const modal = document.querySelector(`.setup`);
```

Comment:
```suggestion
const modalNode = document.querySelector(`.setup`);
```

Хорошая практика использовать слово `node` в нейминге, когда идет работа с DOM узлами. Также в нейминге хорошо упоминать в постфиксе, что возвращает функция или что переменная в себе хранит пр.: `...arr`, `...int`, `...str`, `...obj`. 

Так в будущем будет удобнее с ними работать, понимая из названия что внутри сущности. Такой код легче дебажить и можно избежать глупых ошибок _(JS не строго типизирован, потому часто бывает неочевидное поведение из-за приведения типов)_.